src/OpenLoco/src/GameCommands/RemoveTree.cpp

```diff
@@ -1,27 +1,58 @@
+#include "Audio/Audio.h"
 #include "Economy/Economy.h"
 #include "Economy/Expenditures.h"
 #include "GameCommands.h"
+#include "Map/TileElement.h"
 #include "Map/TileManager.h"
 #include "Map/TreeElement.h"
 #include "Objects/ObjectManager.h"
 #include "Objects/TreeObject.h"
 #include "OpenLoco.h"
 #include "S5/S5.h"
+#include "TownManager.h"
 
 using namespace OpenLoco::Interop;
 
 namespace OpenLoco::GameCommands
 {
+    static loco_global<Core::Prng, 0x00525E20> _prng;
+
+    // 0x0048B089
+    void playDemolishTreeSound(const Map::Pos3 loc)
```

Comment:
If this is not needed outside of this file, just declare it static:
```suggestion
    static void playDemolishTreeSound(const Map::Pos3 loc)
```

You can then remove the header definition.